src/lib/locale-node.ts

```diff
@@ -11,9 +11,13 @@ export function getSupportedLocale() {
 }
 
 export async function getUserLocaleWithFallback() {
-	const { locale } = await loadUserData();
-	if ( ! locale || ! isSupportedLocale( locale ) ) {
+	try {
+		const { locale } = await loadUserData();
+		if ( ! locale || ! isSupportedLocale( locale ) ) {
+			return getSupportedLocale();
+		}
+		return locale;
+	} catch ( error ) {
```

Comment:
I had to add this try/catch to pass the test [`src/lib/tests/site-language.test.ts`](https://github.com/Automattic/studio/blob/e82a94c930718953bb0ac490b60997de8d36ef63/src/lib/tests/site-language.test.ts#L38), with the error:

`Failed to load file /path/to/app/appData/App Name/appdata-v1.json: Error: ENOENT: no such file or directory, open '/path/to/app/appData/App Name/appdata-v1.json'`

It turns out that the `loadUserData` is not is rethrowing the ENOENT' errors.

https://github.com/Automattic/studio/blob/e82a94c930718953bb0ac490b60997de8d36ef63/src/storage/user-data.ts#L74-L83

And `isErrnoException` is returning false for this error:

```
Error: ENOENT: no such file or directory, open '/path/to/app/appData/App Name/appdata-v1.json'
    at open (node:internal/fs/promises:639:25)
    at Object.readFile (node:internal/fs/promises:1249:14) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/path/to/app/appData/App Name/appdata-v1.json'
}
```